src/include/partitioning/custom_part_lib.rb

```diff
@@ -293,13 +293,17 @@ def check_ok_fssize(size_k, volume)
 
       if Ops.less_than(size_k, min_size_k)
         # warning message, %1 is replaced by fs name (e.g. Ext3)
-        # %2 is prelaced by a size (e.g. 10 MB)
+        # %2 is replaced by the user inputted size (e.g. 20GB)
```

Comment:
Same as below: Please change "inputted" to "entered". 
Even if it's just a comment, it hurts the eyes. ;-)